src/preferences/dlgpreferencepage.cpp

```diff
@@ -1,8 +1,20 @@
 #include "preferences/dlgpreferencepage.h"
 
+#include "defs_urls.h"
+
+namespace {
+
+const QString manualUrl(MIXXX_MANUAL_URL);
```

Comment:
- This constant seems to be unused?
- Naming violates coding style
- Why QString instead of QUrl?

Reply:
Sorry, this was a leftover from a previous, more complicated attempt to archieve the same.